src/main/java/de/siphalor/amecs/api/AmecsKeyBinding.java

```diff
@@ -14,37 +14,48 @@
 public class AmecsKeyBinding extends KeyBinding {
 	private final KeyModifiers defaultModifiers;
 
-	//for backward compatibility
 	/**
 	 * Constructs a new amecs keybinding. And because how the vanilla key binding works. It is automatically registered.
-	 * <br> See {@link KeyBindingUtils#unregisterKeyBinding(KeyBinding)} for how to unregister it
+	 * <br>
+	 * See {@link KeyBindingUtils#unregisterKeyBinding(KeyBinding)} for how to unregister it
+	 * If you want to set the key's translationKey directly use {@link #AmecsKeyBinding(String, net.minecraft.client.util.InputUtil.Type, int, String, KeyModifiers)} instead
 	 *
-	 * @param id               the id to use
-	 * @param type             the input type which triggers this keybinding
-	 * @param code             the the default key code
-	 * @param category         the id of the category which should include this keybinding
-	 * @param defaultModifiers the default modifiers
+	 * @param id
+	 *            the id to use
+	 * @param type
+	 *            the input type which triggers this keybinding
+	 * @param code
+	 *            the the default key code
+	 * @param category
+	 *            the id of the category which should include this keybinding
+	 * @param defaultModifiers
+	 *            the default modifiers
```

Comment:
Hm, I dislike how these javadocs are formatted now.

I understand that this is subjective but since this is an open project I'd suggest that you keep to the existing standards.

The small whitespace fixes at the bottom of the file are ok though, my former self had a bad style back then ^^

Reply:
Okay I will change the javadocs back to one line per `@param`